clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_utils.py

```diff
@@ -308,3 +308,18 @@ def prepare_reference_b0(in_dwi, in_bval, in_bvec, low_bval=5, working_directory
     )
 
     return out_reference_b0, out_b0_dwi_merge, out_updated_bval, out_updated_bvec
+
+
+def delete_apply_transform(marker, dir_to_del, base_dir, light):
```

Comment:
Is there something specific to `apply_transform` in this function ?
It looks like it's deleting files in passed directories containing the passed marker, but it is very hard to tell what file structure is expected in line 322 when reading the code. A docstring with examples and some unit tests would be helpful I think.